neuland/executables/neulandSim.cxx

```diff
@@ -0,0 +1,143 @@
+#include "FairBoxGenerator.h"
+#include "FairParRootFileIo.h"
+#include "FairPrimaryGenerator.h"
+#include "FairRootFileSink.h"
+#include "FairRunSim.h"
+#include "R3BCave.h"
+#include "R3BNeuland.h"
+#include "TStopwatch.h"
+#include <FairConstField.h>
+#include <G4RunManager.hh>
+#include <G4UserEventAction.hh>
+#include <TG4EventAction.h>
+#include <boost/program_options.hpp>
+#include <iostream>
+#include <string>
+
+namespace po = boost::program_options;
+
+int main(int argc, const char** argv)
+{
+    TStopwatch timer;
+    auto eventNum = 100000;
```

Comment:
Here I don't know if these variables are int, float, double ....

Reply:
Same above. It's always `int`.